### PR TITLE
improve circle ci tests

### PR DESCRIPTION
**Pull request by DominikGuzei**

### CODE REVIEW

**Comment by tmeasday** [comment]
Interesting. Can you explain the rationale in moving the chimp command out of `package.json` into it's own script?

**Comment by DominikGuzei** [comment]
@tmeasday that was done because i ran into several issues with chimp and Node.js 0.10.x that Meteor is using (. So the problem is that if you run Node via `meteor npm run` you are forced to the Meteor node version. if you run it from outside you can have Node.js 5+ installed globally and basically run Meteor and chimp with different versions.

Here are the related Github issues and discussions:
https://github.com/xolvio/chimp/issues/204
https://github.com/xolvio/chimp/issues/201

**Bottom line:** Meteor really has to upgrade to use up-to-date Node.js versions soon. Until then, we cannot use the `package.json` with chimp.

**Comment by tmeasday** [comment]
Makes sense @DominikGuzei.

We just updated the `master` branch to 1.3.4.4, can you please rebase/merge?

PS we now have a `1.4-beta.10` release; if you'd prefer to write this PR against that, and we can merge this in at the same time as updating to 1.4?

**Comment by DominikGuzei** [comment]
damn … that's why you should never use rebase. Just screwed up the whole PR 😢 
Will reopen another one
